includes/Modules/Analytics/Advanced_Tracking/Measurement_Code_Injector.php

```diff
@@ -21,80 +21,55 @@
  */
 final class Measurement_Code_Injector {
 
-	/**
-	 * Holds a list of event configurations to be injected.
-	 *
-	 * @since n.e.x.t.
-	 * @var array
-	 */
-	private $event_configurations;
-
-	/**
-	 * The javascript code that is injected for event tracking.
-	 *
-	 * @since n.e.x.t.
-	 * @var string
-	 */
-	protected $inject_script;
-
 	/**
 	 * Injector constructor.
 	 *
 	 * @since n.e.x.t.
-	 *
-	 * @param array $event_configurations list of measurement events to track.
 	 */
-	public function __construct( $event_configurations ) {
-		$this->event_configurations = wp_json_encode( $event_configurations );
-		$this->inject_script        = <<<INJECT_SCRIPT
-( function() {
-	function matches( el, selector ) {
-	    const matcher =
-		    el.matches ||
-		    el.webkitMatchesSelector ||
-		    el.mozMatchesSelector ||
-		    el.msMatchesSelector ||
-		    el.oMatchesSelector;
-	    if ( matcher ) {
-	        return matcher.call( el, selector );
-	    }
-	    return false;
-	}
-    var eventConfigurations = {$this->event_configurations};
-	var config;
-	for ( config of eventConfigurations ) {
-		const thisConfig = config;
-		document.addEventListener( config.on, function( e ) {
-			var el = e.target;
-			if ( matches( el, thisConfig.selector ) || matches( el, thisConfig.selector.concat( ' *' ) ) ) {
-				var params = {};
-				params['event_category'] = thisConfig.category;
-				gtag( 'event', thisConfig.action, params );
-			}
-		}, true );
-	}
-	}
-)();
-INJECT_SCRIPT;
-	}
-
-	/**
-	 * Returns the injected Javascript code.
-	 *
-	 * @since n.e.x.t.
-	 *
-	 * @return string the injected JavaScript code
-	 */
-	public function get_injected_script() {
-		return $this->inject_script;
-	}
+	public function __construct() {}
```

Comment:
This should be removed, we shouldn't have unused constructors.

Reply:
This issue has been resolved.